interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,67 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+
+//Function do translate from another language to English
+if (!(function_exists('ltox'))) {
```

Comment:
Is there a reason that this function might exist?

Reply:
Maybe in the future someone will make a function with that name.

Reply:
Lets remove all the function_exists calls in this script. If we have duplicate functions, then we want to know and fix it rather than silently not use the function defined in this script.

Reply:
hi @yurige-matrix ,
Also check out the post above to rec removal of all the function_exist() stuff.